src/main/java/org/javarosa/xpath/XPathTypeMismatchException.java

```diff
@@ -22,6 +22,6 @@ public XPathTypeMismatchException() {
     }
 
     public XPathTypeMismatchException(String s) {
-        super("type mismatch \n" + s);
```

Comment:
It's only every useful to developers who can also just look up the exception from the message. I also found that there wasn't a whole lot of consistency in terms of where this exception class was used, some of the places didn't seem like type mismatches.